sw/device/silicon_creator/manuf/lib/provisioning.c

```diff
@@ -199,26 +276,30 @@ static status_t otp_partition_secret2_configure(const dif_otp_ctrl_t *otp) {
 
   TRY(shares_check(share0, share1, kRootKeyShareSizeIn64BitWords));
 
+  TRY(otp_ctrl_testutils_dai_write64(otp, kDifOtpCtrlPartitionSecret2,
```

Comment:
I think you have to process the token before writing it to `otp`. Something like this (although probably nice to add TODO if you end up using `dif_kmac` instead of cryptolib): 

https://github.com/lowRISC/opentitan/blob/514b622ec829311a826b40b0412aff6f796f2415/sw/device/tests/sim_dv/csrng_lc_hw_debug_en_test.c#L123-L159

Reply:
ah right, good catch. Updated / fixed in #18156.